workflow/rocoto/workflow_tasks.py

```diff
@@ -189,6 +184,41 @@ def get_task(self, task_name, *args, **kwargs):
                                  'Valid tasks are:\n' +
                                  f'{", ".join(Tasks.VALID_TASKS)}')
 
+    @staticmethod
+    def tasks_factory(app_config: AppConfig, cdump: str) -> type['Tasks']:
+        '''
+        Generates a new Tasks object of the appropiate type for the net.
+
+        Parameters
+        ----------
+        app_config: AppConfig
+                    Application configuration to create the task list with. Must have
+                    an attribute named 'net' with a value of 'gfs' or 'gefs'.
+
+        cdump: str
+               CDUMP to generate tasks for (currently a RUN stand-in, will be changed
+               soon).
+
+        Returns
+        -------
+        Tasks: New object of the appropriate type for the net.
+        '''
+        if app_config.net in ['gfs']:
```

Comment:
I think this should work:
```python
me = sys.modules[__name__]
class_name = f"{app_config.net.upper()Tasks}"
try:
    return getattr(me, class_name)(app_config, cdump)
except AttributeError:
    raise AttributeError(f"{class_name} is not a valid class in {me}"
```

It would help you everywhere you have `if-else` conditions.

Reply:
I think the better approach would be to just change it to the uwtools Factory. Not sure if we should do that now, or get this in so things can move forward and then refactor the factories later.